JavaAssignment/src/rahulM/Q1.java

```diff
@@ -0,0 +1,36 @@
+//WAP to print sum,multiply,subtract, divide and remainder of two numbers given by user
+package rahulM;
+class Q1
```

Comment:
Here Logic is correct but try to not write static method because when other class extend your class that time child gets might gets compiler error.